docs/data/charts/gauge/ArcPlaygroundNoSnap.js

```diff
@@ -50,9 +50,9 @@ export default function ArcPlaygroundNoSnap() {
         },
       ]}
       renderDemo={(props) => (
-        <Paper
+        <Box
```

Comment:
My initial idea with the `Paper` we to show the bounding box of the chart, because the gauge position adapts relatively to this bounding box.


But it's less disturbing than expected to don't see it

Reply:
That makes sense! But it feels like it's worth a demo specifically to demonstrate this behavior. The Paper background clashes with the docs bg color 😕